uui/components/overlays/Alert.tsx

```diff
@@ -27,10 +27,12 @@ export interface AlertProps extends IHasChildren, IHasCX, IHasRawProps<React.HTM
     onClose?(): void;
     /** An optional icon to show on the left of the alert */
     icon?: Icon;
+    /** Optional to change size, the compact size value '36' and by default its value is '46' */
+    size?: '36' | '48';
```

Comment:
Let's better use this descriptions: 
`/** Component size. If omitted, 48 size will be used.*/`